### PR TITLE
Pronoun tagging for forest\hunting !

### PR DESCRIPTION
**Pull request by KillerDragain**

### CODE REVIEW

**Comment by scribblecrumb** [comment]
You've got some failing tests due to unexpected characters, check through your changes to ensure there aren't any wayward commas or brackets

File: resources/dicts/patrols/forest/hunting/any.json [hunk 1]
```diff
@@ -1465,14 +1465,14 @@
                 "prey": ["very_small"]
             },
             {
-                "text": "s_c doesn't even hesitate before splitting from the patrol to check on the source of the noise. They quickly regret their impulsivity, however, when an angry raccoon bursts from the bush! s_c gets quite the scare, but thankfully the raccoon only wanted to get away from them and quickly runs off back into the undergrowth.",
+                "text": ""s_c doesn't even hesitate before splitting from the patrol to check on the source of the noise. {PRONOUN/s_c/subject/CAP} quickly {VERB/s_c/regret/regrets} {PRONOUN/s_c/poss} impulsivity, however, when an angry raccoon bursts from the bush! s_c gets quite the scare, but thankfully the raccoon only wanted to get away from {PRONOUN/s_c/object} and quickly runs off back into the undergrowth.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/any.json]: Extra " line 1468

**Review by Thlumyn — CHANGES_REQUESTED** [review]

File: resources/dicts/patrols/forest/hunting/any.json [hunk 2]
```diff
@@ -1312,31 +1312,31 @@
...
                 "exp": 20,
                 "weight": 20,
                 "stat_skill": ["CLEVER,1"],
                 "prey": ["large"]
             },
             {
-                "text": "The prey is running terribly today. But s_c has patience, and wisely waits for the perfect opportunity instead of getting frustrated. They come back to camp with a good contribution to the Clan's stores, having had a relaxing day sitting with their thoughts while waiting for the prey to come to them.",
+                "text": "The prey is running terribly today. But s_c has patience, and wisely waits for the perfect opportunity instead of getting frustrated. {PRONOUN/s_c/subject/CAP} come back to camp with a good contribution to the Clan's stores, having had a relaxing day sitting with {PRONOUN/s_c/poss} thoughts while waiting for the prey to come to {PRONOUN/r_c/object}.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/any.json:1339]: tag "comes"

File: resources/dicts/patrols/forest/hunting/any.json [hunk 3]
```diff
@@ -1552,13 +1552,13 @@
         ],
         "fail_outcomes": [
             {
-                "text": "The patrol mostly ignores the Twolegs. As they hunt, they hear a yelp from r_c. They've crawled into a metal Twolegs box, and there's nothing they or the rest of the patrol can do to get them out - eventually the Twolegs notice, and r_c is taken by them.",
+                "text": "The patrol mostly ignores the Twolegs. As they hunt, they hear a yelp from r_c. {PRONOUN/r_c/subject}{VERB/r_c/'ve/'s} crawled into a metal Twolegs box, and there's nothing {PRONOUN/r_c/subject} or the rest of the patrol can do to get {PRONOUN/r_c/object} out - eventually the Twolegs notice, and r_c is taken by {PRONOUN/r_c/object}.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/any.json]: I think that last tag should actually be them, if it's meaning the Twolegs

File: resources/dicts/patrols/forest/hunting/any.json [hunk 4]
```diff
@@ -1702,13 +1702,13 @@
         ],
         "fail_outcomes": [
             {
-                "text": "The patrol mostly ignores the Twolegs. But as they hunt, the cats encounter appealing, delicious smelling mush in little burrows. Some of the patrol investigates, and doors slam down behind them. There's nothing they or the rest of the patrol can do to get them out - eventually the Twolegs notice, and the trapped cats are taken.",
+                "text": "The patrol mostly ignores the Twolegs. As they hunt, they hear a yelp from r_c. {PRONOUN/r_c/subject}{VERB/r_c/'ve/'s} crawled into a metal Twolegs box, and there's nothing {PRONOUN/r_c/subject} or the rest of the patrol can do to get {PRONOUN/r_c/object} out - eventually the Twolegs notice, and r_c is taken by {PRONOUN/r_c/object}.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/any.json]: I think the last them should stay them to refer to the twolegs

File: resources/dicts/patrols/forest/hunting/any.json [hunk 5]
```diff
@@ -2699,18 +2693,17 @@
...
         },
         "weight": 20,
-        "intro_text": "As app1 tears through the forest, stretching their legs on a run, they spot movement out of the corner of their eye.",
-        "decline_text": "They keep going, not wanting to slow down.",
+        "intro_text": "As app1 tears through the forest, stretching {PRONOUN/app1/poss} legs on a run, {PRONOUN/app1/subject} {VERB/app1/spot/spots} movement out of the corner of {PRONOUN/app1/poss} eye.",
         "chance_of_success": 50,
         "success_outcomes": [
             {
-                "text": "With a huge leap, they dart sideways and pull the tail off a taking flight into blackbird the air. A missed catch, but new nest lining?",
+                "text": "With a huge leap, {PRONOUN/app1/subject} {VERB/app1/dart/darts} sideways and pull the tail off a taking flight into blackbird the air. A missed catch, but new nest lining?",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/any.json]: tag "pulls"

File: resources/dicts/patrols/forest/hunting/any.json [hunk 6]
```diff
@@ -2773,7 +2766,7 @@
                 ]
             },
             {
-                "text": "Nothing's going to escape their claws! s_c skids to a halt and then leaps into action, claw just barely catching in the tail of the squirrel they saw and bringing it down. They purr with satisfaction and pad back to r_c, sharing in the excitement of the hunt with them.",
+                "text": "Nothing's going to escape {PRONOUN/r_c/poss} claws! s_c skids to a halt and then leaps into action, claw just barely catching in the tail of the squirrel {PRONOUN/r_c/subject} saw and bringing it down. {PRONOUN/r_c/subject/CAP} {VERB/r_c/purr/purrs} with satisfaction and pad back to r_c, sharing in the excitement of the hunt with them.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/any.json]: tag "pads"

File: resources/dicts/patrols/forest/hunting/any.json [hunk 7]
```diff
@@ -2859,7 +2852,7 @@
         ],
         "fail_outcomes": [
             {
-                "text": "app1 skids to a halt and tries to sprint after it, but the bird they saw escapes their paws. They chatter angrily at it, and it tries to poop on them. Such is life. app2 snickers at them too, but in a lighthearted way, and soon both apprentices are giggling about it as they continue their run.",
+                "text": "app1 skids to a halt and tries to sprint after it, but the bird {PRONOUN/app1/subject} saw escapes {PRONOUN/app1/poss} paws. They chatter angrily at it, and it tries to poop on them. Such is life. app2 snickers at {PRONOUN/app1/object} too, but in a lighthearted way, and soon both apprentices are giggling about it as they continue their run.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/any.json]: "They chatter angrily at it, and it tries to poop on them" could be tagged

File: resources/dicts/patrols/forest/hunting/any.json [hunk 8]
```diff
@@ -3048,24 +3041,24 @@
...
         "weight": 20,
-        "intro_text": "Your patrol comes across a vole nibbling at a seed.",
-        "decline_text": "Your patrol decides to look for other prey.",
+        "intro_text": "The patrol comes across a vole nibbling at a seed.",
+        "decline_text": "The patrol decides to look for other prey.",
         "chance_of_success": 50,
         "success_outcomes": [
             {
-                "text": "r_c drops into a crouch and lightly stalks towards the vole. They pause, waggling their haunches before pouncing. They catch the vole easily and deliver a killing bite to the back of its head.",
+                "text": "r_c drops into a crouch and lightly stalks towards the vole. {PRONOUN/r_c/subject/CAP} {VERB/r_c/pause/pauses}, waggling {PRONOUN/r_c/poss} haunches before pouncing. {PRONOUN/r_c/subject/CAP} {VERB/r_c/catch/catches} the vole easily and deliver a killing bite to the back of its head.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/any.json]: tag "delivers"

File: resources/dicts/patrols/forest/hunting/any.json [hunk 9]
```diff
@@ -3048,24 +3041,24 @@
...
                 "exp": 10,
                 "weight": 5,
                 "prey": ["medium"]
             },
             {
-                "text": "s_c drops into a crouch and has to consciously stop their tail from swaying in excitement as they stalk the prey. Once they think they're close enough, they give their haunches a little wiggle and leap. They catch the vole squarely between their paws and kill it quickly.",
+                "text": "s_c drops into a crouch and has to consciously stop {PRONOUN/s_c/poss} tail from swaying in excitement as {PRONOUN/s_c/subject} {VERB/s_c/stalk/stalks} the prey. Once {PRONOUN/s_c/subject} {VERB/s_c/think/thinks} {PRONOUN/s_c/subject} {VERB/s_c/'re/'s} close enough, {PRONOUN/s_c/subject} {VERB/s_c/give/gives} {PRONOUN/s_c/poss} haunches a little wiggle and leap. {PRONOUN/s_c/subject/CAP} {VERB/s_c/catch/catches} the vole squarely between {PRONOUN/s_c/poss} paws and kill it quickly.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/any.json]: tag "leaps" and "kills"

File: resources/dicts/patrols/forest/hunting/greenleaf.json [hunk 10]
```diff
@@ -2400,7 +2400,7 @@
                 "prey": ["very_small"]
             },
             {
-                "text": "s_c makes a stupid mistake, too intent on winning the fight to respect their enemy's teeth, and it's tripping over their own feet that saves them. They give up, and settle down to wait a safe distance away until the fox leaves of its own accord.",
+                "text": "s_c makes a stupid mistake, too intent on winning the fight to respect {PRONOUN/s_c/poss} enemy's teeth, and it's tripping over {PRONOUN/s_c/poss} own paws that saves {PRONOUN/s_c/object}. {PRONOUN/s_c/subject/CAP} give up, and settle down to wait a safe distance away until the fox leaves of its own accord.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/greenleaf.json]: tag "gives up" and "settles down"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 11]
```diff
@@ -1152,8 +1152,8 @@
             "apprentice": [1, 6]
         },
         "weight": 20,
-        "intro_text": "app1 catches scent of a mouse nearby. But they're by themselves, and the chill of leaf-bare has stripped the forest of its green cloak; Can they manage the catch?",
-        "decline_text": "Your patrol ignores the mouse.",
+        "intro_text": "app1 catches scent of a mouse nearby. But {PRONOUN/app1/subject}{VERB/app1/'re/'s} by {PRONOUN/app1/SELF}, and the chill of leaf-bare has stripped the forest of its green cloak; Can {PRONOUN/app1/subject} manage the catch?",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: why is self capitalized?

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 12]
```diff
@@ -1292,7 +1292,7 @@
                 ]
             },
             {
-                "text": "As the patrol settles in, tracking mouse scent mostly, s_c wanders back, and as they put a paw down the ground squeaks. They lean down, jaws snapping, and meet the eyes of the rest of the patrol with an insolent smirk.",
+                "text": "As the patrol settles in, tracking mouse scent mostly, s_c wanders back, and as {PRONOUN/s_c/subject} {VERB/s_c/put/puts} a paw down the ground squeaks. {PRONOUN/s_c/subject/CAP} {VERB/s_c/lean/leans} down, jaws snapping, and meet the eyes of the rest of the patrol with an insolent smirk.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "meets"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 13]
```diff
@@ -1439,13 +1439,13 @@
                 "prey": ["medium"]
             },
             {
-                "text": "Though it tries to escape, r_c is faster and smarter - they slam a paw down on the squirrel's tail before it can go very far, and kill it with ease.",
+                "text": "Though it tries to escape, r_c is faster and smarter - {PRONOUN/s_c/subject} {VERB/s_c/slam/slams} a paw down on the squirrel's tail before it can go very far, and kill it with ease.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "kills"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 14]
```diff
@@ -1454,7 +1454,7 @@
         ],
         "fail_outcomes": [
             {
-                "text": "r_c breaks a twig as they ready themself to pounce, and the resulting crack alerts the squirrel. It doesn't even spare them a glance as it scurries up the tree and out of sight.",
+                "text": "r_c breaks a twig as {PRONOUN/r_c/subject} {VERB/r_c/ready/readies} {PRONOUN/r_c/self} to pounce, and the resulting crack alerts the squirrel. It doesn't even spare them a glance as it scurries up the tree and out of sight.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: "It doesn't even spare them a glance as it scurries up the tree and out of sight" you can tag "them" here

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 15]
```diff
@@ -1474,26 +1474,26 @@
...
         },
         "weight": 20,
-        "intro_text": "app1 scents a squirrel on the air! Leaf-bare will make this a difficult hunt, and apprehension prickles down their spine as they weigh their options.",
+        "intro_text": "app1 scents a squirrel on the air! Leaf-bare will make this a difficult hunt, and apprehension prickles down {PRONOUN/app1/poss} spine as {PRONOUN/app1/subject} {VERB/app1/weigh/weighs} {PRONOUN/app1/poss} options.",
         "decline_text": "They decide to move on and ignore the squirrel for now.",
         "chance_of_success": 10,
         "success_outcomes": [
             {
-                "text": "app1 slides into a hunting pose, trying not to focus too hard on making it perfect. They're right not to - their catch is flawless, and they return home with their chest puffed out in pride.",
+                "text": "app1 slides into a hunting pose, trying not to focus too hard on making it perfect. {PRONOUN/app1/subject}{VERB/app1/'re/'s} right not to - {PRONOUN/app1/poss} catch is flawless, and {PRONOUN/app1/subject} return home with {PRONOUN/app1/poss} chest puffed out in pride.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "returns"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 16]
```diff
@@ -1514,12 +1514,12 @@
             "normal adult": [1, 6]
         },
         "weight": 20,
-        "intro_text": "p_l instructs app1 to scent the air, and nods in approval when app1 says they smell squirrel. Though they don't want app1 to feel pressured, p_l makes sure they know how valuable a leaf-bare squirrel would be to their Clanmates before asking what they want to do.",
+        "intro_text": "p_l instructs app1 to scent the air, and nods in approval when app1 says {PRONOUN/app1/subject} {VERB/app1/smell/smells} squirrel. Though {PRONOUN/app1/subject} {VERB/app1/do/does}n't want app1 to feel pressured, p_l makes sure {PRONOUN/app1/subject} {VERB/app1/know/knows} how valuable a leaf-bare squirrel would be to {PRONOUN/app1/poss} Clanmates before asking what {PRONOUN/app1/subject} {VERB/app1/want/wants} to do.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: I think those 2nd and 3rd tags should be p_l not app1

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 17]
```diff
@@ -1648,7 +1648,7 @@
                 "prey": ["very_small"]
             },
             {
-                "text": "The patrol fans out, some cats climbing, some staying on the ground. The hunt comes to an abrupt end, however, when r_c yowls in fear as they plummet from the branches and land heavily on the ground. Their Clanmates help them home and settle them into the medicine cat's den, guilt and regret prickling at them for pursuing such a foolhardy hunt.",
+                "text": "The patrol fans out, some cats climbing, some staying on the ground. The hunt comes to an abrupt end, however, when r_c yowls in fear as {PRONOUN/r_c/subject} {VERB/r_c/plummet/plummets} from the branches and land heavily on the ground. {PRONOUN/r_c/poss/CAP} Clanmates help {PRONOUN/r_c/object} home and settle {PRONOUN/r_c/object} into the medicine cat's den, guilt and regret prickling at them for pursuing such a foolhardy hunt.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "lands"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 18]
```diff
@@ -1807,13 +1807,13 @@
         ],
         "fail_outcomes": [
             {
-                "text": "p_l is silently furious at r_c when they find them. A snowstorm is no time to go haring off by themselves, and it's destroyed the patrol's hunting chances.",
+                "text": "p_l is silently furious at r_c {PRONOUN/p_l/subject} {VERB/p_l/find/finds} {PRONOUN/r_c/object}. A snowstorm is no time to go haring off by {PRONOUN/r_c/self}, and it's destroyed the patrol's hunting chances.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: put "when" back

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 19]
```diff
@@ -2163,26 +2163,26 @@
...
+                "text": "p_l drops into a hunting crouch, watching the oblivious cardinal as it perches on a low and barren shrub branch. With a wriggle of {PRONOUN/p_l/poss} hind legs, p_l leaps, snagging the cardinal in {PRONOUN/p_l/poss} front paws and killing it with a quick bite to the neck.",
                 "exp": 15,
                 "weight": 20,
                 "prey": ["small"]
             },
             {
-                "text": "p_l watches the cardinal, waiting for the right moment to strike, when it suddenly flits off of the low-hanging branch it was perched on. Without thinking, p_l leaps into the air and grabs the cardinal mid-flight with their front paws. They pin it down and kill it with a quick bite to the neck.",
+                "text": "p_l watches the cardinal, waiting for the right moment to strike, when it suddenly flits off of the low-hanging branch it was perched on. Without thinking, p_l leaps into the air and grabs the cardinal mid-flight with {PRONOUN/p_l/poss} front paws. {PRONOUN/p_l/subject} {VERB/p_l/pin/pins} it down and kill it with a quick bite to the neck.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "kills"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 20]
```diff
@@ -2163,26 +2163,26 @@
...
                 "exp": 15,
                 "weight": 5,
                 "prey": ["small"]
             }
         ],
         "fail_outcomes": [
             {
-                "text": "p_l drops into a hunting crouch, watching the cardinal peck at some invisible bit of food. Suddenly, it flies off, and despite p_l's superb reflexes, they miss the cardinal and fall muzzle-first into the snow. Mouse-dung!",
+                "text": "p_l drops into a hunting crouch, watching the cardinal peck at some invisible bit of food. Suddenly, it flies off, and despite p_l's superb reflexes, {PRONOUN/p_l/subject} {VERB/p_l/miss/misses} the cardinal and fall muzzle-first into the snow. Mouse-dung!",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "falls"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 21]
```diff
@@ -2202,12 +2202,12 @@
...
+        "decline_text": "app1 decides not to pursue the bird. {PRONOUN/app1/subject/CAP} {VERB/app1/have/has} never hunted one alone before, and the loud crunching of the snow would surely make {PRONOUN/app1/poss} hunt more difficult than necessary.",
         "chance_of_success": 55,
         "success_outcomes": [
             {
-                "text": "app1 carefully measures their pawsteps as they drop into a hunting crouch behind a barren shrub and approach the robin that is pecking at the icy ground before them. With a waggle of their haunches, they pounce and kill the bird with a purr of pride.",
+                "text": "app1 carefully measures {PRONOUN/app1/poss} pawsteps as {PRONOUN/app1/subject} {VERB/app1/drop/drops} into a hunting crouch behind a barren shrub and approach the robin that is pecking at the icy ground before {PRONOUN/app1/object}. With a waggle of {PRONOUN/app1/poss} haunches, {PRONOUN/app1/subject} {VERB/app1/pounce/pounces} and {VERB/app1/kill/kills} the bird with a purr of pride.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "approaches"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 22]
```diff
@@ -2241,12 +2241,12 @@
             "apprentice": [1, 6]
         },
         "weight": 20,
-        "intro_text": "p_l leads app1 through the snow-covered forest, showing them how to distribute their weight across their paws to avoid sinking and to move silently. Suddenly, p_l flicks their tail: they have spotted a bird up ahead.",
-        "decline_text": "app1 shakes their head when p_l asks if they'd like to try their paws at hunting a bird. They're just getting used to navigating in the snow quietly; surely catching a bird will be too difficult at the moment.",
+        "intro_text": "p_l leads app1 through the snow-covered forest, showing {PRONOUN/app1/object} how to distribute {PRONOUN/app1/poss} weight across {PRONOUN/app1/poss} paws to avoid sinking and to move silently. Suddenly, p_l flicks {PRONOUN/p_l/poss} tail: {PRONOUN/app1/subject} {VERB/app1/have/has} spotted a bird up ahead.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: I think those last 2 tags should be p_l, not app1

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 23]
```diff
@@ -2241,12 +2241,12 @@
...
+        "decline_text": "app1 shakes {PRONOUN/app1/poss} head when p_l asks if {PRONOUN/app1/subject}'d like to try {PRONOUN/app1/poss} paws at hunting a bird. {PRONOUN/app1/subject/CAP} {VERB/app1/'re/'s} just getting used to navigating in the snow quietly; surely catching a bird will be too difficult at the moment.",
         "chance_of_success": 55,
         "success_outcomes": [
             {
-                "text": "p_l silently demonstrates the right way to crouch and stalk in this unfamiliar wintry world. app1 mimics them faithfully, and springs at the starling from two fox-lengths away. It doesn't know what hits it, and in a heartbeat, it is dangling from app1's jaws. p_l gives a purr of approval.",
+                "text": "p_l silently demonstrates the right way to crouch and stalk in this unfamiliar wintry world. app1 mimics {PRONOUN/app1/object} faithfully, and springs at the starling from two fox-lengths away. It doesn't know what hits it, and in a heartbeat, it is dangling from app1's jaws. p_l gives a purr of approval.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: should be p_l, not app1

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 24]
```diff
@@ -2305,33 +2305,33 @@
...
         "chance_of_success": 35,
         "success_outcomes": [
             {
-                "text": "r_c presses on through the blinding snow, letting their ears guide them. A faint scent arrives at their nose before long - grouse! The storm obscures r_c as they drop into a hunting crouch and spring on the unwary bird, hidden as it is under a layer of snow. The blizzard passes before nightfall, and r_c makes their way home, catch in tow.",
+                "text": "r_c presses on through the blinding snow, letting {PRONOUN/r_c/poss} ears guide {PRONOUN/r_c/object}. A faint scent arrives at {PRONOUN/r_c/poss} nose before long - grouse! The storm obscures r_c as {PRONOUN/r_c/subject} {VERB/r_c/drop/drops} into a hunting crouch and spring on the unwary bird, hidden as it is under a layer of snow. The blizzard passes before nightfall, and r_c makes {PRONOUN/r_c/poss} way home, catch in tow.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "springs"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 25]
```diff
@@ -3658,7 +3658,7 @@
                 "prey": ["very_small"]
             },
             {
-                "text": "s_c makes a stupid mistake, too intent on winning the fight to respect their enemy's teeth, and it's tripping over their own feet that saves them. They give up, and settle down to wait a safe distance away until the fox leaves of its own accord.",
+                "text": "s_c makes a stupid mistake, too intent on winning the fight to respect their enemy's teeth, and it's tripping over {PRONOUN/s_c/poss} own feet that saves {PRONOUN/s_c/object}. {PRONOUN/s_c/subject/CAP} {VERB/s_c/give/gives} up, and settle down to wait a safe distance away until the fox leaves of its own accord.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "settles"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 26]
```diff
@@ -3744,17 +3744,17 @@
         },
         "weight": 20,
         "intro_text": "r_c catches scent of a mouse nearby, one that would be greatly appreciated for the fresh-kill pile.",
-        "decline_text": "Your patrol ignores the mouse.",
+        "decline_text": "tHE patrol ignores the mouse.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: Capitalization

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 27]
```diff
@@ -3930,7 +3930,7 @@
                 ]
             },
             {
-                "text": "As the patrol settles in, tracking mouse scent mostly, s_c wanders back, and as they put a paw down the ground squeaks. They lean down, jaws snapping, and meet the eyes of the rest of the patrol with an insolent smirk.",
+                "text": "As the patrol settles in, tracking mouse scent mostly, s_c wanders back, and as {PRONOUN/s_c/subject} {VERB/s_c/put/puts} a paw down the ground squeaks. {PRONOUN/s_c/subject} {VERB/s_c/lean/leans} down, jaws snapping, and meet the eyes of the rest of the patrol with an insolent smirk.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "meets"

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 28]
```diff
@@ -4020,7 +4020,7 @@
                 ]
             },
             {
-                "text": "These are the times that it really pays to have a hunter of s_c's caliber on the patrol - they really go above and beyond, impressing everyone and leaving a trail of terror among the local mice.",
+                "text": "These are the times that it really pays to have a hunter of s_c's caliber on the patrol - {PRONOUN/s_c/subject} really go above and beyond, impressing everyone and leaving a trail of terror among the local mice.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: tag "goes"

File: resources/dicts/patrols/forest/hunting/newleaf.json [hunk 29]
```diff
@@ -2820,12 +2820,12 @@
...
         "weight": 20,
-        "intro_text": "Your patrol catches the scent of a fox - but is it red, or gray?",
-        "decline_text": "Your patrol decides not to pursue the fox.",
+        "intro_text": "The patrol catches the scent of a fox - but is it red, or gray?",
+        "decline_text": "The patrol decides not to pursue the fox.",
         "chance_of_success": 40,
         "success_outcomes": [
             {
-                "text": "The stocky gray vixen they track down smells of milk, and though she's near double r_c's weight and all of it muscle, they meet her eyes squarely, and leap to attack. c_n does not tolerate sharing their territory with foxes.",
+                "text": "The stocky gray vixen they track down smells of milk, and though she's near double r_c's weight and all of it muscle, {PRONOUN/r_c/subject} {VERB/r_c/meet/meets} her eyes squarely, and leap to attack. c_n does not tolerate sharing {PRONOUN/r_c/poss} territory with foxes.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/newleaf.json]: tag "leaps"

File: resources/dicts/patrols/forest/hunting/newleaf.json [hunk 30]
```diff
@@ -3159,7 +3159,7 @@
                 ]
             },
             {
-                "text": "As the patrol settles in, tracking mouse scent mostly, s_c wanders back, and as they put a paw down the ground squeaks. They lean down, jaws snapping, and meet the eyes of the rest of the patrol with an insolent smirk.",
+                "text": "As the patrol settles in, tracking mouse scent mostly, s_c wanders back, and as {PRONOUN/s_c/subject} {VERB/s_c/put/puts} a paw down the ground squeaks. {PRONOUN/s_c/subject/CAP} {VERB/s_c/lean/leans} down, jaws snapping, and meet the eyes of the rest of the patrol with an insolent smirk.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/newleaf.json]: tag "meets"

File: resources/dicts/patrols/forest/hunting/newleaf.json [hunk 31]
```diff
@@ -3399,7 +3399,7 @@
         ],
         "fail_outcomes": [
             {
-                "text": "While rounds a bend in the path, app1 hears a bird call. Jerking {PRONOUN/app1/poss} head up, they lose {PRONOUN/app1/poss} footing and skid down a slope that's more mud than grass. {PRONOUN/app1/subject/CAP}{VERB/app1/'re/'s} not that badly hurt, but app1 is tender enough after that fall to give up and go home.",
+                "text": "While rounds a bend in the path, app1 hears a bird call. Jerking {PRONOUN/app1/poss} head up, {PRONOUN/app1/subject} {VERB/app1/lose/loses} {PRONOUN/app1/poss} footing and skid down a slope that's more mud than grass. {PRONOUN/app1/subject/CAP}{VERB/app1/'re/'s} not that badly hurt, but app1 is tender enough after that fall to give up and go home.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/newleaf.json]: tag "skids"

**Review by Thlumyn — CHANGES_REQUESTED** [review]
There's a lot of spots that say {PRONOUN/r_c/subject} {VERB/r_c/'re/'s} - I can't remember off the top of my head, but we might need to remove the space in between those.

**Comment by KillerDragain** [comment]
> There's a lot of spots that say {PRONOUN/r_c/subject} {VERB/r_c/'re/'s} - I can't remember off the top of my head, but we might need to remove the space in between those.

Upon searching every file there appeared to be only be a couple. Went ahead and fixed those up!

File: resources/dicts/patrols/forest/hunting/leaf-bare.json (hunk 11 above)
Comment (KillerDragain) [reply resources/dicts/patrols/forest/hunting/leaf-bare.json]: i just got really excited /silly

File: resources/dicts/patrols/forest/hunting/leaf-bare.json [hunk 32]
```diff
@@ -1807,13 +1807,13 @@
...
                 "exp": 0,
                 "weight": 20,
                 "prey": ["very_small"]
             },
             {
-                "text": "This is bad. When they find r_c, there's barely any warmth left in their pelt, and p_l can't wake them. They're left with an impossible decision - leave r_c there in the snow, or risk everyone else. p_l chooses r_c, and the warmth of their Clanmates curling around them saves r_c's life, though none of the patrol can escape the consequences of the cold.",
+                "text": "This is bad. When {PRONOUN/p_l/subject} {VERB/p_l/find/finds} r_c, there's barely any warmth left in {PRONOUN/r_c/poss} pelt, and p_l can't wake {PRONOUN/r_c/object}. {PRONOUN/p_l/subject/CAP} {VERB/p_l/'re/'s} left with an impossible decision - leave r_c there in the snow, or risk everyone else. p_l chooses r_c, and the warmth of {PRONOUN/r_c/POSS} Clanmates curling around {PRONOUN/r_c/object} saves r_c's life, though none of the patrol can escape the consequences of the cold.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/hunting/leaf-bare.json]: Looks like the pronouns tests were failing here - POSS shouldn't be capitalized

**Review by Thlumyn — CHANGES_REQUESTED** [review]

**Review by Thlumyn — APPROVED** [review]

**Pull request merged**